cmake/UUtils.cmake

```diff
@@ -1,22 +1,21 @@
-message(STATUS "LOOKING FOR UUTILS")
-find_package(UUtils 2.0.2 COMPONENTS base hash debug QUIET)
+find_package(UUtils 2.0.4 COMPONENTS base hash debug QUIET)
 
 if (UUtils_FOUND)
-  message(STATUS "Found UUtils: ${uutils_DIR}")
+  message(STATUS "Found UUtils: ${UUtils_DIR}")
 else(UUtils_FOUND)
     message(STATUS "Failed to find UUtils, going to compile from source.")
     if (FIND_FATAL)
-      message(FATAL_ERROR "Failed to find doctest with CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}")
+      message(FATAL_ERROR "Failed to find UUtils with CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}")
     endif(FIND_FATAL)
     include(FetchContent)
     set(UUtils_WITH_TESTS OFF CACHE BOOL "UUtils tests")
     set(UUtils_WITH_BENCHMARKS OFF CACHE BOOL "UUtils benchmarks")
     FetchContent_Declare(
             UUtils
-            GIT_REPOSITORY https://github.com/mikucionisaau/UUtils.git
-            GIT_TAG cmake-alias
-            #GIT_REPOSITORY https://github.com/UPPAALModelChecker/UUtils.git
-            #GIT_TAG v2.0.2
+            #GIT_REPOSITORY https://github.com/mikucionisaau/UUtils.git
+            #GIT_TAG cmake-alias
```

Comment:
Please remove these

Reply:
I can surely remove, but it is very handy to have for testing ;-)